src/status_im2/contexts/activity_center/events.cljs

```diff
@@ -153,6 +157,74 @@
   [cofx notification]
   (notifications-reconcile cofx [(assoc notification :read true)]))
 
+(defn- update-read-flag
+  [notifications read?]
+  (reduce (fn [acc notification]
+            (conj acc (assoc notification :read read?)))
+          []
+          notifications))
+
+
+;;;; Mark all notifications as read
```

Comment:
comment is redundant imo, consider removing 👍

Reply:
Removed 👍